lib/charms/opensearch/v0/opensearch_secrets.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""In this class we manage certificates relation.
+
+This class handles certificate request and renewal through
+the interaction with the TLS Certificates Operator.
+
+This library needs https://charmhub.io/tls-certificates-interface/libraries/tls_certificates
+library is imported to work.
+
+It requires a charm that extends OpenSearchBaseCharm as it refers internal objects of that class.
+— update_config: to disable TLS when relation with the TLS Certificates Operator is broken.
+"""
+
+import logging
+
+from charms.opensearch.v0.constants_charm import Scope
+from charms.opensearch.v0.constants_tls import CertType
+from charms.opensearch.v0.opensearch_internal_data import SecretsDataStore
+from ops.charm import ActionEvent
+from ops.framework import Object
+
+# The unique Charmhub library identifier, never change it
+LIBID = "8bcf275287ad486db5f25a1dbb26f920"
```

Comment:
Yes, now this is a quesion for me, thx v much for pointing it out.

How do we deal with these constants? 

Also LIBVERSION, and so, in case of charms...?

Reply:
`LIBPATCH` should be increment whenever you update the LIB, so you have a new revision of the library that can be uploaded to Charmhub. It's like the minor version.

`LIBAPI` is like the major version (for breaking changes).

You can keep both as they are right now, as this is the first revision of the library.

Reply:
New `LIBID` was generated by `charmcraft`